### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Top Operations_ScreenReader: NVDA is announcing the column number and column number sequence incorrectly.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/azuredatastudio#20784]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #ScreenReader; #WCAG4.2.3; #FTP; #NVDA

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:
1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Tab to "Top operations" control from the right navigation and tab to any of the data cell.
7. Observe that NVDA is announcing the column number and column number sequence incorrectly.

### Actual:
NVDA is announcing the column number and column number sequence incorrectly.

****NOTE:** Similar issue is observed throughout the application.**

### Expected Result:
NVDA should announce the column number correctly like it should start from the first column i.e. from the left side and also the sequence should be correct it should not be random.

### Attachment:
![A11y_Azure Data Studio_Toolbar_Top Operations_ScreenReader](https://user-images.githubusercontent.com/115216656/194823045-35020894-8da0-4217-8c4d-12a13b0b16f7.gif)

**Comment by aasimkhan30** [comment]
This issue is fixed in the latest insiders. Please verify and close it

**Comment by Rama8345** [comment]
Verified the issue in latest insider build. The issue is fixed. So, closing the bug. Please find below attachment for reference.
![20784_Closed](https://user-images.githubusercontent.com/115216656/204991403-74d7b635-1772-43ce-a873-0ce9cd265139.gif)

**Comment by Rama8345** [comment]
Verified the issue in latest insiders build. The issue is fixed. So, closing the bug. Please find below attachment for reference.
![20784_Closed](https://user-images.githubusercontent.com/115216656/204992864-418f34e2-09e1-4874-b85d-ce627bdc0811.gif)
